test/common/persistence/KeyValuePersistenceTest.php

```diff
@@ -19,20 +19,32 @@
 
 namespace oat\generis\test\common\persistence;
 
-class KeyLargeValuePersistenceTest extends \PHPUnit_Framework_TestCase
+use \PHPUnit_Framework_TestCase as TestCase;
+
+class KeyValuePersistenceTest extends TestCase
```

Comment:
Launching this test case with Redis driver provides me the following errors in PHPUnit:

```
1) oat\generis\test\common\persistence\KeyValuePersistenceTest::testDelExistsLarge
Failed asserting that true is false.

/home/jerome/Development/TAO/taoplatform300/generis/test/common/persistence/KeyValuePersistenceTest.php:78

2) oat\generis\test\common\persistence\KeyValuePersistenceTest::testDelExists
Failed asserting that true is false.

/home/jerome/Development/TAO/taoplatform300/generis/test/common/persistence/KeyValuePersistenceTest.php:89

3) oat\generis\test\common\persistence\KeyValuePersistenceTest::testMapMapControl
Failed asserting that true is false.

/home/jerome/Development/TAO/taoplatform300/generis/test/common/persistence/KeyValuePersistenceTest.php:78
/home/jerome/Development/TAO/taoplatform300/generis/test/common/persistence/KeyValuePersistenceTest.php:110
```

Reply:
I just commited something to make it easier to test with other drivers.

Reply:
Solved, inMemoryDriver is not persistent, Redis is. It's why in the second test, the driver already have the key 'test'